### ISSUE TITLE
Crate a group file for the offline reconstruction

### ISSUE THREAD

**Issue by fwyzard** [fwyzard/circles#10]

**Comment by slava77** [comment]
does the file format support patterns/regular expressions for the module instance names?

**Comment by fwyzard** [comment]
Yes, it should be possible to use glob patterns (`?` and `*`) for both type names and module names.

If you need regexp I could look for a way to add them as well, but so far I assumed glob patterns would be enough.
